### PR TITLE
[Notification System] Add notification customization screens

### PR DESCRIPTION
**Pull request by SukiCai**
# Notification Customization 

Requires: 
N/A

## Subtasks / Progress
- [ ] Adjustment of the screens relationship

## Testing
- [x] Manually Testing

### CODE REVIEW

File: mobile/src/stateMachine/views/notificationCustomize/notificationsCustomize.tsx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    switchStyles: {
+        paddingHorizontal: 3,
+    },
+    buttonView : {
+        alignContent: 'center',
+        alignItems: 'center',
+        padding: 5
+     },
+    insturctionsButton: {
+        width: 320,
```
Comment (MarkDavies0) [inline mobile/src/stateMachine/views/notificationCustomize/notificationsCustomize.tsx:179]: Change this width to a percentage (eg, 80%) so it scales nicely on different screen sizes

**Review by MarkDavies0 — COMMENTED** [review]
Great work so far! Most of my comments are pretty minor and some are simply style related

File: mobile/src/constants/images/app.ts [hunk 2]
```diff
@@ -55,6 +55,9 @@ import MoneyIconSVG from 'src/assets/images/app/money.svg';
 import PhysicalIconSVG from 'src/assets/images/app/physical.svg';
 import SelfEsteemIconSVG from 'src/assets/images/app/selfesteem.svg';
 import SleepIconSVG from 'src/assets/images/app/sleep.svg';
+import NotificationScreen1 from 'src/assets/images/app/Notification-Instructions-1.png';
```
[the remark this answers is not in the document]
Comment (SukiCai) [reply mobile/src/constants/images/app.ts:58]: Hi Mark, I've tried to use svg before, but the image not shown when I test it, I saw we have png in our image base so I used png instead, do you think I should insist to use svg?

[the remark this answers is not in the document]
Comment (MarkDavies0) [reply mobile/src/constants/images/app.ts:58]: SVGs have worked for me (see SleepIconSVG etc.) but I have found that if you are trying to set the color using style you need to [change the stroke to "currentColor"](https://github.com/bipolarbridges/companion-app/pull/84/files#diff-9b9a51cad9bc3755c71b221eee20ed623bd8748f058cf86cf754ebd9322e8233). So I would try adjusting with the stroke values. Your issue could be that the stroke color is the same as the view's background color

[the remark this answers is not in the document]
Comment (SukiCai) [reply mobile/src/constants/images/app.ts]: Hi Mark I found that the SVG not shown the problem is caused by the source type conflict in preferenceInstruction.tsx and it seems like png would be preferred, do you have any ideas to fix this?

[the remark this answers is not in the document]
Comment (MarkDavies0) [reply mobile/src/constants/images/app.ts]: Hi Suki I am not sure which icons you are trying to use exactly but I have always used svgs like this:
`<Images.backIcon` where backIcon is your svg. I noticed you added
    `// notification
    notificationScreen1: NotificationScreen1,
    notificationScreen2: NotificationScreen2,
    notificationScreen3: NotificationScreen3,` to app.ts. Are these the icons you are trying to use?

[the remark this answers is not in the document]
Comment (SukiCai) [reply mobile/src/constants/images/app.ts]: Oh, actually, they are not icons, they are images that shown and switched in the instructions

[the remark this answers is not in the document]
Comment (MarkDavies0) [reply mobile/src/constants/images/app.ts]: Ah okay, PNGs probably best then

**Comment by MarkDavies0** [comment]
Waiting on #82 before fixing conflicts

**Comment by MarkDavies0** [comment]
See #132

**Pull request closed without merging**